loama/src/components/header/HeaderContextMenu.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+    <aside>
+        <span>{{ store.usedPod }}</span>
+        <div>
+            <label for="pod">Selected Pod</label>
+        <select>
+            <option v-for="pod in pods" :key="pod">{{ pod }}</option>
+        </select>
+        </div>
+        <a @click.prevent="logout">
+            <PhSignOut/>
+            <span>Sign out</span>
+        </a>
+    </aside>
+</template>
+
+<script setup lang="ts">
+import router from '@/router';
+import { store } from '@/store';
+import type { Session } from '@inrupt/solid-client-authn-browser';
+import { PhSignOut } from '@phosphor-icons/vue';
+import { listPods } from 'loama-controller';
+const pods = await listPods(store.session as Session);
+
+async function logout() {
+    store.session.logout();
+    router.push('/'); 
+}
+</script>
+
+<style lang="css" scoped>
+aside {
+    border-radius: 0.5rem;
+    display: flex;
+    flex-flow: column nowrap;
+    border: 0.25rem solid var(--solid-purple);
+    background-color: var(--off-white);
+}
+aside > div {
```

Comment:
why no scss?

Reply:
We decided to only use CSS instead of SCSS within the team. It's actually has a lot of features that are already present in SCSS and I don't really see why we should use it rn

Reply:
nw just asking a question. I prefer the structure it provides but if you don't care its fine by me